libs/logging/src/log_documentation.test.ts

```diff
@@ -27,7 +27,7 @@ describe('test cdf documentation generation', () => {
     expect(structuredData.DeviceModel).toEqual('VxAdmin 1.0');
     expect(structuredData.GeneratedDate).toEqual('2020-07-24T00:00:00.000Z');
     expect(structuredData.EventTypeDescription).toHaveLength(5);
-    expect(structuredData.EventIdDescription).toHaveLength(58);
+    expect(structuredData.EventIdDescription).toHaveLength(53);
```

Comment:
I completely removed the old `smartcard-programmed-override-write-protection` log, and the other 4 smart card programming logs are now emitted by the VxAdmin backend instead of the VxAdmin frontend (what this test covers)